kayak_core/src/styles.rs

```diff
@@ -15,7 +25,7 @@ where
     T: Default + Clone,
 {
     fn default() -> Self {
-        Self::Default
+        Self::Unset
     }
```

Comment:
If we don't want `Style::default` to return `StyleProp::Unset` fields, we could replace this with a `Style::unset()` method. But I prefer implementing it on `Default` since it allows us to call `unwrap_or_default()` with the unset properties.